Ops handbook hr management (#834)
publiccodenet/about: activities/hr-management/salary-administration.md

---
type: Guide
explains: how salary administration is managed
---

# Salary administration

* Tentoo takes care of our salary administration. Tentoo’s account is linked to ExactOnline, no need to submit anything related to salary administration
* [Login via website](https://tentoo.nmbrs.nl/applications) or the app
* The Chief Executive and Operations Coordinator have ‘company’ logins with which they can access the personnel files and keep them up to date

> * All employees have their own login as well as access to the app (can be set to English and to Dutch). They can submit expenses, download salary stubs, request vacation days and check their vacation time balance. They can also fill in mutation forms for new addresses, partners, email addresses (we use the personal email for this tool, since the salary stubs etc. have to remain accessible) and emergency contact info

Reviewer: ```suggestion
* All employees have their own login as well as access to the app (can be set to English and to Dutch). They can submit expenses, download salary stubs, request vacation days and check their vacation time balance. They can also fill in mutation forms for new addresses, partners, email addresses (we use the personal email for this tool, since the salary stubs etc. have to remain accessible) and emergency contact info
```
```suggestion
* All employees have their own login as well as access to the app (can be set to English and to Dutch). They can submit expenses, download salary stubs, request vacation days and check their vacation time balance. They can also fill in change notification (mutation) forms for new addresses, partners, emergency contact details, and email addresses (we use the personal email for this tool, since the salary stubs etc. have to remain accessible)
```

* Expenses have to be approved in the portal. The Chief Executive and Operations Coordinator can do this

> * Every month, before the 15th of the month, the Operations Coordinator opens the salary run. Any mutations need to be send in before the run is started

Reviewer: ```suggestion
* Every month, before the 15th of the month, the Operations Coordinator opens the salary run. Any change notifications (mutations) need to be sent in before the run is started
```

> * Mutations are e.g. more or less hours, different pay, reimbursements. Forms are available in the Tentoo NMBRS dashboard and are send from there as well

Reviewer: ```suggestion
* Change notification (mutations) include more or less hours, different pay, or reimbursements. Forms are available in the Tentoo NMBRS dashboard and are send from there as well
```

> * Change notification (mutations) include more or less hours, different pay, or reimbursements. Forms are available in the Tentoo NMBRS dashboard and are send from there as well

Reviewer: ```suggestion
* Change notifications (mutations) include more or less hours, different pay, or reimbursements. Forms are available in the Tentoo NMBRS dashboard and are send from there as well
```

* Once the salary run has finished the Operations Coordinator receives a notification
* In the Tentoo NMBRS dashboard the latest SEPA file can be found. The date of the file needs to be changed to the date we want to pay (24 or 25th of the month)

> * The Operations Coordinator uploads the SEPA to the bank and processes it

Reviewer: Does the OC or the bank process it?

* When ready, the Operations Coordinator sends a mail to the Chief Executive to release the payments. Make sure to include a PDF with an overview of the salaries, which can also be downloaded from the Tentoo NMBRS dashboard

> ## Loonheffing (LH) – payroll tax

Reviewer: Why is the LH acronym included? It doesn't look like it's used anywhere else.

* Tentoo reports monthly “loonheffingen en sociale verzekeringspremies” (wage and social taxes) to the tax office
* Tentoo sends an email to the Operations Coordinator with amount due

> * The Operations Coordinator sets-up the payment in the banking environment, assigns the payment date (e.g. March needs to be paid in April) and completes it with the first signature

Reviewer: ```suggestion
* The Operations Coordinator sets up the payment in the banking environment, assigns the payment date (e.g. March needs to be paid in April) and completes it with the first signature
```
